rules/common/private/utils.bzl

```diff
@@ -193,9 +201,35 @@ def action_singlejar(
     ctx.actions.run(
         arguments = [args],
         executable = ctx.executable._singlejar,
-        execution_requirements = resolve_execution_reqs(ctx, {"supports-workers": "0"}),
+        execution_requirements = resolve_execution_reqs(
+            ctx,
+            {
+                "supports-workers": "0",
+                "supports-path-mapping": "1",
+            },
+        ),
         mnemonic = _SINGLE_JAR_MNEMONIC,
         inputs = all_inputs,
         outputs = [output],
         progress_message = progress_message,
     )
+
+def separate_src_jars(srcs):
```

Comment:
Where is this used?

Reply:
I lost a part of my changes. Good catch, I'll go actually use this. We do this all over the repo, which is why I made a function for it. On some branch of mine I actually used this, but I must have copied part of it over, but not all.